lib/activerecord-bitemporal/bitemporal.rb

```diff
@@ -354,8 +366,22 @@ def each_association(
       using EachAssociation
 
       def _create_record(attribute_names = self.attribute_names)
+        current_time = Time.current
+
         # 自身の `valid_from` を設定
-        self.valid_from = valid_datetime || Time.current if self.valid_from == ActiveRecord::Bitemporal::DEFAULT_VALID_FROM
+        self.valid_from = valid_datetime || current_time if self.valid_from == ActiveRecord::Bitemporal::DEFAULT_VALID_FROM
+
+        self.transaction_from = current_time if self.transaction_from == ActiveRecord::Bitemporal::DEFAULT_TRANSACTION_FROM
+
+         # Assign only if defined created_at and deleted_at
+        if has_column?(:created_at)
+          self.transaction_from = self.created_at if changes.key?("created_at")
```

Comment:
In the case (`created_at` was changed), it seems `valid_from` and `transaction_from` is **not** the same values.
Is this valid records? Does this cause problems?

Reply:
yes, `valid_from` and `transaction_from` are not the same because they have different information.
And they are the same by default.

```ruby
company = Company.create
pp company.created_at.iso8601(10)
# => "2020-11-04T08:18:03.1952580000Z"
pp company.transaction_from.iso8601(10)
# => "2020-11-04T08:18:03.1952580000Z"
pp company.valid_from.iso8601(10)
# => "2020-11-04T08:18:03.1952580000Z"
```